src/parser/deserializer.ts

```diff
@@ -334,8 +345,11 @@ function deserializeObject(
         (buffer[index++] << 16) |
         (buffer[index++] << 24);
       const long = new Long(lowBits, highBits);
+      if (useBigInt64) {
+        value = BigInt(lowBits) | (BigInt(highBits) << 32n);
+      }
```

Comment:
Can we instead lean on the native API here, create and ask a DataView to getBigInt64 at this offset. I want to avoid the peculiarities of translating bit shifting to bigint creation. The logic you went with looks perfectly correct (+ tests agree) 

Also I never came up with a way to test this effectively, its something a linter should do, but I couldn't find an eslint rule off the bat, we want to avoid shipping `n` notation inside the BSON code 1. we don't need it, 2. those who aren't in environments that support it won't crash on import, instead they may crash on use. If we weren't changing to use DataView we'd want to change `<< 32n` to `<< BigInt(32)`. This requirement is called out in the design doc